scripts/maps.js

```diff
@@ -201,11 +206,21 @@ function rollOffVenue() {
     setMarkers(null);
 }
 
-function createVenueList(venues) {
+function resetList(list) {     
+    var child = list.lastElementChild;  
+    while (child) { 
```

Comment:
Is this the most optimal way to remove list items? It looks like it goes through and removes each list item in turn, which would lead to a redraw of the page for every item. Is there a parent element we can empty instead? Or remove and then reinstate the parent element?

Reply:
I'm sure there is a better way, I tried setting the element to null at the end but couldn't seem to make it work. Happy to have another look in a few days, have been at it for a while.

Reply:
Found jQuery .empty(), will update. (Or can use.innerHTML="")